### ISSUE TITLE
Conflict with Highlighted References

### ISSUE THREAD

**Issue by stesvis** [hicknhack-software/semantic-colorizer#9]
I believe there is a conflict with the highlighted colors.
This extension seems to override the foreground colors. I chose white foreground for highlighted definition and reference, but the SEM variable color takes priority and makes it hard to read.

**Comment by arBmind** [comment]
@stesvis Sorry, that you have trouble with the plugin.

I have trouble to grasp what you mean. Can you please add a screenshot, to show what you mean.

**Comment by stesvis** [comment]
No problem.
So i set Highlighted Definition with white foreground and green background
![highlight](https://cloud.githubusercontent.com/assets/6062684/23267806/14245a28-f9a9-11e6-9a9f-21a07fc92b3d.PNG)

But then when I go to the code, I see that the foreground is purple (which is quite hard to read
![editor](https://cloud.githubusercontent.com/assets/6062684/23267841/2a8e9602-f9a9-11e6-82c2-5671be5daf0d.PNG)

Which is the foreground color that I set in Semantic Local
![sem-local](https://cloud.githubusercontent.com/assets/6062684/23267873/448c6804-f9a9-11e6-8a47-3cdecd0832ad.PNG)

So I believe that Sematic Local is overriding the foreground color that I set in Highlighted Definition.
In the same way, it overrides the foreground I set in Highlighted Reference/Current Line etc. etc.
![image](https://cloud.githubusercontent.com/assets/6062684/23267992/a23ff39e-f9a9-11e6-9534-fce7d2d9c747.png)

^^^ that one should be with a white foreground as well. Other than that I love the plugin!

Thanks for your reply!

**Comment by arBmind** [comment]
@stesvis Thank you for the in depth explanation. I was able to reproduce it on my setup here.

I did an experiment with the experimental instance of visual studio. It runs only the stock extensions. The Highlight Foreground Color does not override any of the default colors.

So I guess this is a general issue with Visual Studio.

If anybody has an idea how we can work around this I would be happy to implement it.
